demos/python_demos/action_recognition/action_recognition.py

```diff
@@ -88,27 +91,37 @@ def main():
     ie = IECore()
 
     if 'MYRIAD' in args.device:
-        myriad_config = {"VPU_HW_STAGES_OPTIMIZATION": "YES"}
-        ie.set_config(myriad_config, "MYRIAD")
+        myriad_config = {'VPU_HW_STAGES_OPTIMIZATION': 'YES'}
+        ie.set_config(myriad_config, 'MYRIAD')
 
     if args.cpu_extension and 'CPU' in args.device:
-        ie.add_extension(args.cpu_extension, "CPU")
+        ie.add_extension(args.cpu_extension, 'CPU')
 
-    decoder_target_device = "CPU"
+    decoder_target_device = 'CPU'
     if args.device != 'CPU':
         encoder_target_device = args.device
     else:
         encoder_target_device = decoder_target_device
 
     encoder_xml = args.m_encoder
-    encoder_bin = args.m_encoder.replace(".xml", ".bin")
-    decoder_xml = args.m_decoder
-    decoder_bin = args.m_decoder.replace(".xml", ".bin")
-
+    encoder_bin = args.m_encoder.replace('.xml', '.bin')
     encoder = IEModel(encoder_xml, encoder_bin, ie, encoder_target_device,
                       num_requests=(3 if args.device == 'MYRIAD' else 1))
-    decoder = IEModel(decoder_xml, decoder_bin, ie, decoder_target_device, num_requests=2)
-    video_demo(encoder, decoder, videos, args.no_show, args.utilization_monitors, args.fps, labels)
+
+    if args.m_decoder is not None:
+        decoder_xml = args.m_decoder
+        decoder_bin = args.m_decoder.replace('.xml', '.bin')
+        decoder = IEModel(decoder_xml, decoder_bin, ie, decoder_target_device, num_requests=2)
+        decoder_seq_size = decoder.input_size[1]
```

Comment:
You should make sure that the `--seq` option was not specified, so that the user isn't led to believe that it does something when it does not.

Reply:
These arguments are in a mutually exclusive group. Isn't it enough?

Reply:
Oh, my bad. Didn't realize you did that.